cmac/cmac_radar.py

```diff
@@ -157,14 +156,26 @@ def cmac(radar, sonde, config, geotiff=None, flip_velocity=False,
     radar.add_field('gate_id', my_fuzz,
                     replace_existing=True)
 
-    if 'ground_clutter' or 'clutter' in radar.fields.keys():
+    if 'ground_clutter' in radar.fields.keys():
         # Adding fifth gate id, clutter.
         clutter_data = radar.fields['ground_clutter']['data']
-        gate_data = radar.fields['gate_id']['data']
+        gate_data = radar.fields['gate_id']['data'].copy()
         radar.fields['gate_id']['data'][clutter_data == 1] = 5
         notes = radar.fields['gate_id']['notes']
         radar.fields['gate_id']['notes'] = notes + ',5:clutter'
         radar.fields['gate_id']['valid_max'] = 5
+    if 'classification_mask' in radar.fields.keys():
+        clutter_data = radar.fields['classification_mask']['data']
+        gate_data = radar.fields['gate_id']['data'].copy()
+        radar.fields['gate_id']['data'][clutter_data == 8] = 5
+        radar.fields['gate_id']['data'][clutter_data == 16] = 5
+        radar.fields['gate_id']['data'][clutter_data == 4] = 0
+        radar.fields['gate_id']['data'][clutter_data == 1] = 0
+        radar.fields['gate_id']['data'][clutter_data == 2] = 0
+        radar.fields['gate_id']['data'][gate_data == 0] = 0
+        notes = radar.fields['gate_id']['notes']
+        radar.fields['gate_id']['notes'] = notes + ',5:clutter'
+        radar.fields['gate_id']['valid_max'] = 5
```

Comment:
Going to have to trust you on this

Reply:
@scollis 
So the classification mask has flags as such:
 'flag_masks': array([ 1,  2,  4,  8, 16], dtype=int32),
 'flag_meanings': 'second_trip third_trip interference clutter sunspoke'
I was going to revisit this and ask so glad you reminded me, but I added 1, 2, 4 to multi trip and clutter to 5

Reply:
![class_mask](https://user-images.githubusercontent.com/19153455/94304636-7b960400-ff35-11ea-9eba-da50ef3d2968.png)

Reply:
Interference is that also multi trip? It looks like multi trip in the scan:
![cmac_four_panel_plot cacticsaprcmacppi c1 20181111 010003](https://user-images.githubusercontent.com/19153455/94304756-ad0ecf80-ff35-11ea-80d0-76c487aed788.png)